src/elm/Page/Dashboard/Claim.elm

```diff
@@ -87,7 +95,13 @@ view ({ shared } as loggedIn) model =
                         Page.fullPageLoading
 
                     Loaded claim ->
-                        div [ class "bg-white py-2" ]
+                        let
+                            isCurrentUserValidator =
+                                claim.action.validators
+                                    |> List.any
+                                        (\v -> v.account == loggedIn.accountName)
+                        in
+                        div [ class "bg-gray-100 py-2" ]
```

Comment:
@amiskov the spacing on the feedback still looks weird, looks like its because of this line, adding the `py-2` class is causing trouble:

<img width="1492" alt="Screen Shot 2020-10-22 at 12 53 03" src="https://user-images.githubusercontent.com/1082127/96898425-136c0c80-1466-11eb-9420-d7e319d2e154.png">

Reply:
Oops, didn't notice this last time... `py-2` did the trick, thanks!